docs/MANUAL.md

```diff
@@ -203,7 +203,7 @@ The other major hurdle for productionizing (but one well worth the effort) is ob
 1. At this point, you should have certificates available at `/etc/letsencrypt/live/example.com` (replacing `example.com` with your domain). You should serve `fullchain.pem` as the certificate and `privkey.pem` as its private key. However, these files are owned by root and the private key is not readable by the ergo role user, so you won't be able to use them directly in their current locations. You can write a post-renewal hook for certbot to make copies of these certificates accessible to the ergo role user. For example, install the following script as `/etc/letsencrypt/renewal-hooks/post/install-ergo-certificates`, again replacing `example.com` with your domain name, and chmod it 0755:
 
 ````bash
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
`env` loads bash from $PATH and also works on systems that don't come with global `bash` such as FreeBSD(?) assuming it's installed somewhere.

Reply:
I'm not sure about all the implications of this change. These answers indicate that for system scripts, it may be preferable to invoke the "normal" interpreter under `/bin` or `/usr/bin`:

* https://unix.stackexchange.com/a/29620
* https://stackoverflow.com/a/21614603

I'd feel better if we didn't use `/usr/bin/env` here?

Reply:
I use `/usr/bin/env bash` for scripts too, would just like to add that I don't think the possible drawbacks are a concern here, but would be interested to see how am wrong on this...and the higher compatibility is a plus. Also like using /usr/bin/env for python below.

Reply:
As a practical example, I have a shell access on `FreeBSD 12.3-RELEASE r371126 GENERIC` where there is no bash. All my scripts broke on that, until I did, probably unorthodox symlink, `~/.local/bin/bash -> /usr/local/bin/zsh`

I understand that also macOS ships with zsh instead of bash, and while I don't have hard feelings on this, this is at least two scenarios where `/bin/bash` shebang is undesirable in my opinion.

Reply:
Would it not be better to change it to `/bin/sh` to address those concerns? It doesn't look like there are any bash-isms, any posix compatible shell should work.

Reply:
>I understand that also macOS ships with zsh instead of bash,

AIUI MacOS ships with both, it's just that zsh is now the default. Users can run `chsh -s /bin/bash` to switch back to bash without installing any software.